marathon_quiz.cpp

```diff
@@ -93,6 +93,9 @@ float newScore(int,int);
 //Nickname creation function prototype definition
 void nicknameCreation(char*);
 
+//Funciton for display quetions page
+void displayQuestionsInPages(Question* questionHead)
```

Comment:
@Starfisheunyoung Please add the missing semicolon at the end of the function displayQuestionsInPages